pandas/plotting/_matplotlib/tools.py

```diff
@@ -392,6 +392,11 @@ def handle_shared_axes(
             row_num = lambda x: x.rowNum
             col_num = lambda x: x.colNum
 
+        if compat.mpl_ge_3_4_0:
+            is_first_col = lambda x: x.get_subplotspec().is_first_col()
```

Comment:
This gives the following error in some builds:

```
>   is_first_col = lambda x: x.get_subplotspec().is_first_col()
E   AttributeError: 'SubplotSpec' object has no attribute 'is_first_col'
```

But that seems to happen in an environment with older matplotlib. So something might be wrong with the version check?

Reply:
Looking at the code mpl_ge_3_4_0 might be a function not a boolean? So statement should be compat.mpl_ge_3_4_0()?

Reply:
Good catch, thanks.  updated